docker-compose.yml

```diff
@@ -22,6 +22,12 @@ services:
       # We don't set up ssl certs for redis local dev
       REDIS_SSL_VALIDATION: none
       REDIS_URL: redis://db
+      AWS_REGION: us-east-1
```

Comment:
We should add a comment linking to the part in the localstack docs that explains that these are the defaults